src/devices/namron.ts

```diff
@@ -1624,4 +1624,48 @@ export const definitions: DefinitionWithExtend[] = [
             m.temperature({reporting: undefined}),
         ],
     },
+    {
+        zigbeeModel: ["4512791"],
+        model: "4512791",
+        vendor: "Namron AS",
+        description: "Namron Simplify Zigbee dimmer (1/2-polet / Zigbee / BT)",
+
+        // Modern extend (on/off, brightness, transition, power_on_behavior)
+        extend: [extend.light_onoff_brightness()],
```

Comment:
```suggestion
        extend: [m.light(), m.electricityMeter()],
```

`fromZigbee`, `exposes`, `endpoint` and `configure` can be removed.